src/clj/test/cards/ice.clj

```diff
@@ -408,6 +408,43 @@
       (take-credits state :corp 2)
       (is (= 5 (:current-strength (refresh lotus))) "Lotus Field strength increased"))))
 
+(deftest mausolus
+  ;; Mausolus - 3 adv tokens change the subroutines
+  (do-game
+    (new-game (default-corp [(qty "Mausolus" 1)])
+              (default-runner[(qty "NetChip" 5)]))
+    (play-from-hand state :corp "Mausolus" "HQ")
+    (let [mau (get-ice state :hq 0)]
+      (core/rez state :corp mau)
+      (take-credits state :corp)
+      (run-on state :hq)
+      (is (= (:credit (get-corp)) 3) "corp starts encounter with 3 crs")
+      (is (= (count (:discard (get-runner))) 0) "runner starts encounter with no cards in heap")
+      (is (= (:tag (get-runner)) 0) "runner starts encounter with 0 tags")
+      (card-subroutine state :corp mau 0)
+      (card-subroutine state :corp mau 1)
+      (card-subroutine state :corp mau 2)
+      (is (= (:credit (get-corp)) 4) "corp ends encounter with 4 crs")
+      (is (= (count (:discard (get-runner))) 1) "runner ends encounter with 1 card in heap")
+      (is (= (:tag (get-runner)) 1) "runner ends encounter with 1 tag")
```

Comment:
Maybe these `is` comments could be rephrased to refer to the Mausolus subroutines? E.g. "corp gained 1 credit from Mausolus".